integration_tests/sdk/delete_workflow_test.py

```diff
@@ -136,17 +138,23 @@ def test_delete_workflow_saved_objects_twice(client):
         flow_2_id = flow_list[1]
```

Comment:
for line 130, aren't the two flow ids going to be exactly the same? It should be `num_runs=2` then, and we probably should just track a single flow ids to be less confusing. You can just check that the flow id returned by both `run_flow_test()` calls is the same, so you don't have to call `list_saved_objects()` twice with the exact same arguments.

Reply:
For this specific test, I'm checking the successful deletion case and unsuccessful deletion case works as expected. To test this, I have two workflows that write to the same table. When I delete the table in the first workflow, it is successful but when I delete it in the second workflow, it is unsuccessful because the table has already been deleted.

Reply:
If it's helpful, I can add the message above as a comment above the test function.

Reply:
ah gotcha - yeah can we add a comment? It's not super clear that the name of the workflow is being randomized, which is different from the above test